ldclient/client.py

```diff
@@ -149,10 +149,11 @@ def __init__(self, sdk_key, config=None, start_wait=5):
         log.info("Waiting up to " + str(start_wait) + " seconds for LaunchDarkly client to initialize...")
         update_processor_ready.wait(start_wait)
 
-        if self._update_processor.initialized:
+        if self._update_processor.initialized() is True:
```

Comment:
This was not evaluating correctly.. it would indicate success even when the streaming processor had not been initialized.